content/posts/2020-08-15-foundation-announcement.kr.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "발표: 도지코인 재단의 재설립"
+date = "2021-08-16"
+[ author ]
+  name = "Dogecoin Foundation"
++++
+
+크나큰 기쁨속에  **도지코인 재단** *(설립 2014)*의 재설립을 발표합니다, 다시 회복한 집중으로 도지코인 이코 시스템, 커뮤니티의 지원, 도지코인 블록체인의 미래를 홍보할 것 입니다.
+
+### 초점
+
+도지코인 재단은 준수 도지코인 지갑의 기본 참조 구현으로 남아있는 '도지코인 코어' 프로젝트를 장악하기 위한 것이 아니다. 
+
+도지코인 재단은 도지코인 에코 시스템의 확장을 고려하고 있으며, 다가오는 주 중에, 도지코인 코어 지갑의 보다 더 빠른 적용 그리고  도지코인을 도입하고 싶어하는 금융, 소셜, 자선 프로젝트 프로젝트를 위한 쉬운 API를 위해 프로젝트를 발표할 것이다. 우리는 도지코인의  성공은 세계적인 도입과 이용에 있다고 생각하며, 우리의 집중은 도지코인의 도입을 풀뿌리 수준(grassroots level)에서 도울 프로젝트에 있다.
+
+### 도지코인 브랜드 보호
+
+수년간, 도지코인 커뮤니티는 사람들에게서 최선를 믿어왔고 이름과 로고에 대하여 불상사가 없길 기대했습니다. 도지코인은 커뮤니티(the people)의 것임이 당연하다.
+
+하지만, 비교적 최근에, 다양한 관할 구역에서 '도지코인' 브랜드의 등록을 하기 위한 시도들이 있었으며, 그 방식은 커뮤니티와 프로젝트가 더 이상 그 브랜드를 쓰지 못하게 할 방식이었다.
+
+모르는 이들이 많을 것이지만, 우리는 굉장히 좋은 시베(shibe)를 칭찬하고 싶습니다: Jens Wiechers, 2014년 부터 혼자, 개인적인 손실을 많이 봐가며 '도지코인 재단'의 이름 아래서 모든이 들이 도지코인 브랜드를 사용할 수 있게끔 이런 저작권 공격과 싸워왔습니다. 
+
+재단은 도지코인 마크와 로고를 가지고 있으며 커뮤니티를 위하여 유지할 것 입니다. 이는 재단이 계속해서 도지코인 브랜드를 보호 할 수있으며 (굉장히 자유로운 라이센스 아래서) 도지코인 관련 밈, 프로잭트, 그리고 재미를 위하여 허용할 수 있을 것이다. — 우리를 믿어도 좋습니다.
+
+### 도지코인 성명서
+
+저희는 도지코인의 의미를 고민해보았습니다: 기술적인 프로젝트으로서 그리고 전세계의 선(good)을 위한 운동으로서. 저희가 도지코인 성명서라 부르기로한 4가지의 문장으로 요약할 수 있었습니다. 두개는 기술에 관한것이며 두개는 사회관련 입니다.
+
+저희 는 함께한다면, 'Do Only Good Everyday'의 정수를 담고 있는 이 문장들은 우리가 앞으로 선택을 할때와 도지코인 에코시스템의 발전을 위해서 좋은 나침반이 될 것이라고 생각합니다.
+
+[도지코인 성명서](/manifesto)
```

Comment:
one little suggestion: if you want you can use `/kr/manifesto` so users can directly see korean version of the manifesto